scss/share-component.scss

```diff
@@ -26,3 +26,10 @@
 		}
 	}
 }
+
+// Override default framework styles
+.field {
+	.chzn-container {
+		display: inline-block;
+	}
+}
```

Comment:
You also don't need to override this style back to it's original state. Here you're overriding 'display: inline-block' with 'display: inline-block'...